### ISSUE TITLE
R - overlapping between the the training and testing datasets

### ISSUE THREAD

**Issue by ramkrisp** [facebookincubator/prophet#95]
Hello all,

My data contained 54 months where I split it into 51/3 in order to model on the first 51 data points and test the prediction of the next 3 months and the remaining data points using monthly model.

Here is my code for the prediction:
`future <- make_future_dataframe(m, periods = 3, freq = "m")`
And used the following to forecast:
`fcst <- predict(m, future)`

However it seems that there is one month missing/overlapping (getting 53 points instead of 54):
> tail(future)
           ds
48 2016-06-01
49 2016-07-01
50 2016-08-01
51 2016-09-01
52 2016-10-01
53 2016-11-01`

Any suggestion?

Thanks,
Rami

**Comment by bletham** [comment]
I think this should be the same issue as in #82, which was just fixed in 1d37f7f. Could you install from Github and check if it is working now?
```
devtools::install_github('facebookincubator/prophet', subdir='R')
```

**Comment by ramkrisp** [comment]
Yup, it was fixed, thanks!
